packages/sucrase/package.json

```diff
@@ -66,5 +67,12 @@
       "!**/recipes/**",
       "!**/types.ts"
     ]
-  }
+  },
+  "contributors": [
+    {
+      "name": "Vinayak Kulkarni",
+      "email": "inbox.vinayak@gmail.com",
+      "url": "https://vinayakkulkarni.dev"
+    }
+  ]
```

Comment:
Please remove this block. I understand that you'd like credit for the change, but we don't add individual contributors to plugins' `package.json` files. You'll be recognized here: https://github.com/rollup/plugins/graphs/contributors and our `LICENSE` references that page specifically.